game-backend/src/map/maps/warpedArena.ts

```diff
@@ -0,0 +1,65 @@
+import { Position } from '@darwin/types';
+
+const warpedArena: Position[] = [
+  // left
+  { x: 3, y: 2 },
```

Comment:
General Note: What about a dedicated string format for these?

```
.......oo.......
.......oo.......
```

The import won't be much more difficult but the readability would be improved.
Sorry, to drop your line count though 😅

Reply:
I second that ^^.